src/components/BookItem.js

```diff
@@ -1,12 +1,21 @@
+/* eslint-disable camelcase */
```

Comment:
- [ ] It would be nice if you don't disable your linters and try to fix the errors manually because linters prevent you from introducing the errors in your code that can break your app completely 😨 So kindly remove this line and fix the error manually 👍